practice_app/apps/user/templates/user-detail.html

```diff
@@ -78,10 +78,10 @@ <h4 class="alert-heading">Failed</h4>
                             <input type="submit" name="submit" value="Get">
                             <br>
                     </form>
-                </div>	
+                </div>
             </div>
         </div>
 	</div>
     <a href="{%url 'user-home' %}"> <p style="text-align:center">Back to Home Page</p></a>
```

Comment:
As I understood, you also have to change this. Currently, user-api is not working properly.

Reply:
user-home to user:user-home